lib/perl/Genome/Model/ClinSeq.pm

```diff
@@ -645,33 +645,18 @@ sub _resolve_workflow_for_build {
     }
 
     #TophatJunctionsAbsolute - Run tophat junctions absolute analysis on normal
-    if ($build->normal_rnaseq_build) {
-        my $normal_tophat_junctions_absolute_op = $self->tophat_junctions_absolute_op($workflow, 'normal');
-    }
-    #TophatJunctionsAbsolute - Run tophat junctions absolute analysis on tumor
-    my $tumor_tophat_junctions_absolute_op;
-    if ($build->tumor_rnaseq_build) {
-        $tumor_tophat_junctions_absolute_op = $self->tophat_junctions_absolute_op($workflow, 'tumor');
-    }
-
     #CufflinksExpressionAbsolute - Run cufflinks expression absolute analysis on normal
     if ($build->normal_rnaseq_build) {
+        my $normal_tophat_junctions_absolute_op = $self->tophat_junctions_absolute_op($workflow, 'normal');
```

Comment:
These cufflinks steps ran successfully in a test ClinSeq model.  Are you suggesting that both variables should be removed since they are not used elsewhere?

>  7118929   execute  succeeded  2016-03-31T11:03:49       00:02:14           . Performing cufflinks expression absolute analysis for normal sample
>   7118933   execute  succeeded  2016-03-31T11:03:51       00:02:23           . Performing cufflinks expression absolute analysis for tumor sample